lib/tasks/heart_rate_data.rake

```diff
@@ -0,0 +1,9 @@
+namespace :heart_rate_data do
+  desc "Update Fitbit heart rate data for the last user"
+  task sync: :environment do
+    user = User.last
+    user.heart_rate_datas << HeartRateData.new
+
+    user.heart_rate_datas.last.update_heart_rate_data!
```

Comment:
heh, I'm old. I didn't know rails had to persist new objects now; I thought it was in memory only until you explicitly called `save` on it. Live and learn.

RE: a factory method: I agree with the point of view and how to get there. This is simple enough for now and we can see how the app evolves. See the TODO checkbox I added to the PR description here please. I think moving this code into the user class will make a refactor later both simpler and more obvious to do.